atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/TracingCassandraClient.java

```diff
@@ -71,6 +75,10 @@ public CassandraClient delegate() {
             sink.accept("consistency", consistency_level.name());
```

Comment:
Maybe log something here as well? This will only get logged IFF the trace is observable, if it's missing we know they're basically not getting published.

Reply:
The trace is not initialised at this point, so not sure what to log.

Reply:
Log anything, just a constant message is fine. You'll know whether the trace was published to consumers (i.e. logged) or not. If this is logged, and we don't see the trace, we know something really weird is going on.

Reply:
added